src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -29,15 +29,15 @@ public sealed class SystemConformanceProvider
 
         public SystemConformanceProvider(
             IModelInfoProvider modelInfoProvider,
-            ISearchParameterDefinitionManager searchParameterDefinitionManager,
+            SearchableSearchParameterDefinitionManagerResolver searchParameterDefinitionManagerResolver,
```

Comment:
Is there a reason to switch this from taking in an Interface to taking in the concrete class?

Reply:
Oh, I see this was discussed above :)